ui/css/perf.css

```diff
@@ -219,6 +219,39 @@ to { opacity: 0; }
     overflow: hidden;
 }
 
+/*Test Chooser graph*/
+
+.modal-body {
+  display: flex;
+  display: -weblit-flex;
```

Comment:
I believe should be `-webkit..` here, and it looks like there are two `display` properties instead of one?